platform/brains2/src/duo/main.cpp

```diff
@@ -373,10 +373,12 @@ static void main_init(AudioAmplifier& headphone_preamp, AudioAmplifier& speaker_
   audio_init();
   AudioInterrupts();
 
-  MIDI.setHandleStart(sequencer_restart);
-  usbMIDI.setHandleStart(sequencer_restart);
-  MIDI.setHandleContinue(sequencer_restart);
-  usbMIDI.setHandleContinue(sequencer_restart);
+  MIDI.setHandleStart(sequencer_start_from_MIDI);
+  usbMIDI.setHandleStart(sequencer_start_from_MIDI);
+
+  MIDI.setHandleContinue(sequencer_start_from_MIDI);
+  usbMIDI.setHandleContinue(sequencer_start_from_MIDI);
+
```

Comment:
Should we actually respond the same to `Start` and `Continue`? Or should we restart the sequencer from the first step on `Start` messages?

Reply:
yes, in case of the DUO a `Start` and `Continue` should mean the same